web/.gitignore

```diff
@@ -31,6 +31,4 @@ yarn-error.log*
 .env.production.local
 
 # vercel
-.vercel
-yarn.lock
-package-lock.json
+.vercel
```

Comment:
Better to add a new line.

Reply:
Already have a new line.

Reply:
Why we need a new line?

Reply:
Because GitHub warn it.

Reply:
Maybe we can add a tool to add it. 
But for now, I think it's OK. Because other projects don't have this requirement either.

Reply:
ok